src/libraries/skinDynLib/test/skinContactTest.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2017 Fondazione Istituto Italiano di Tecnologia (IIT)
+ * CopyPolicy: Released under the terms of the GNU GPL v2.0.
+ */
+
+#include <cassert>
+#include <cmath>
+#include <ctime>
```

Comment:
I don't think they are the same. `std::chrono::system_clock::now()` is the wall time, while `std::clock()` is the process time.